e2e/tests/ui-driven/src/context.ts

```diff
@@ -327,3 +329,29 @@ async function deleteTeam(adminGQLClient: GraphQLClient, context: Context) {
     }
   }
 }
+
+async function setupGovPaySecret($admin: CoreDomainClient, context: Context) {
```

Comment:
A little bit inelegant maybe but this was the best approach I could find to set up this value in the DB!